docs/host/host.md

```diff
@@ -33,30 +33,102 @@ Cordoning a node marks it as unschedulable. This feature is useful for performin
+
+
+### 2. Check volume status
+
+- Go to the `Volume` page of Longhorn GUI.
+- Check all volumes are healthy.
+
+### 3. Evict replicas from the deleting node
+
+- Go to the **Node** page of the Longhorn GUI.
+- Select the deleting node, click the icon on the right side, and click **Edit node and disks**.
+- Set **Node Scheduling** to `Disable` and **Evict Requested** to `True`. Click the **Save** button.
+- Wait until replicas on the node become 0. You can go to the **Node** page of the Longhorn GUI, and check the **Replicas** count of the deleting node becomes `0`.
+
+:::note
+
+Note if there are not enough nodes in the cluster to accept the replicas from the deleting node, the eviction can never finish. In this case, you need to consider taking the risk to let some volumes stay degraded until you add more nodes to the cluster.
+
+:::
+
+### 4. Deal with non-migratable VMs
+
+Some VMs can't live-migrate. You need to handle them accordingly:
+
+- VMs with PCI passthrough devices or vGPU devices
+    
+    A PCI device is bound to a node, you need to remove the PCI device from the VM, or delete the VM and recreate a new VM from a backup/snapshot.
 
-You will lose all data of the control plane node after deleting the RKE2 service.
+- VMs with a node selector or affinity rules that bind the VMs to the deleting node
+
+    You need to edit the VM and update the node selector or affinity rules.
+
+- VMs with a VLAN network that binds the VMs to the deleting node
+
+    You need to edit the VM and change the VLAN network.
+
+In general, for a non-migratable VM, it's recommended to have a backup first and you can restore the backup to new VMs in the future.
+
+### 5. Evict workloads out of the deleting node
+
+You can manually live-migrate VMs (see [Live Migration](../vm/live-migration.md)) on the deleting node to other nodes.
+
+If your cluster runs a Harvester version newer than `v1.1.2` (inclusive), you can also set the deleting node to maintenance mode (see [Node Maintenance](./host.md#node-maintenance)) to automatically live-migrate VMs and workload out of the node. You can confirm all workloads on the deleting node are evicted successfully by checking a node's state is set to `Maintenance`:
+
+![node-maintain-completed.png](/img/v1.3/host/node-maintain-completed.png)
+
+:::note
+
+If there are only two control plane nodes left in the cluster, Harvester doesn't allow setting a node to maintenance mode. You need to manually drain the node with the command:
+
+```
+kubectl drain <node_name> --force --ignore-daemonsets --delete-local-data --pod-selector='app!=csi-attacher,app!=csi-provisioner'
+```
+
+Please replace `node_name` with the deleting node. Again, removing a control plane node when there are only two control plane nodes left in the cluster is **not recommended**. Please use it at your own risk.
 
 :::
```

Comment:
```suggestion
Again, removing a control plane node in this situation is **not recommended** because etcd data is not replicated. Failure of a single node can cause etcd to lose its quorum and shut the cluster down.
:::
```